### DISCUSSION TITLE
I'm working on an Android platform, and I can't find the codemagic.yaml file.

### DISCUSSION THREAD

**Discussion by orot-shindonghwi** [codemagic-ci-cd/codemagic-docs#2346] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [ ] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [x] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Android (Kotlin)

### Steps to reproduce

I'm working on an Android platform, and I can't find the codemagic.yaml file.

### Expected results

I'm working on an Android platform, and I can't find the codemagic.yaml file.

It is a private project, and the entire project is structured in the Monorepo structure as follows

<img width="416" alt="image" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/109847226/501e53c1-d87b-4228-9782-9c0807e4fc36">

root
    - - codemagic.yaml
    - - project1
    - - project2

There are no typos.

### Actual results

<img width="1097" alt="스크린샷 2024-02-01 오후 2 55 39" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/109847226/bb3cc647-6659-4f3b-919e-0a55a9e9e7d6">

**Comment by himesh-cm** [comment]
Hi @orot-shindonghwi,

Please note `codemagic.yaml` file must be created in the root directory of the repository. By default, the working directory is the root of the repository.

In order to target apps inside your monorepo app, working_directory key is used. For example, the following sample snippet shows how it works:

```
workflows:
  default-workflow:
    name: Default workflow
    # Specify path to the app folder like this
    working_directory: my_first_app

```

Let me know, if you face any issues.

**Comment by himesh-cm** [comment]
Hi @orot-shindonghwi,
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
